src/types/ScholarshipEligibility.test.ts

```diff
@@ -0,0 +1,27 @@
+import ScholarshipEligibility from './ScholarshipEligibility';
+
+test('construct default', () => {
```

Comment:
You could add similar tests as these for the converter tests in src/models/Scholarships.test.tsx instead.